squire_web/src/header.rs

```diff
@@ -0,0 +1,41 @@
+use yew::prelude::*;
+use yew_router::prelude::*;
+
+use crate::Route;
+
+#[function_component(Header)]
+pub fn header() -> Html {
+    let nav = use_navigator().unwrap();
+    let make_route = move |route| {
+        let nav = nav.clone();
+        Callback::from(move |_| nav.push(&route))
+    };
+    html! {
+        <header>
+            <nav class="navbar navbar-expand-md navbar-dark bg-dark">
+                <div class="container-fluid">
+                    <a class="navbar-brand" href="#">{ "Squire Web" }</a>
```

Comment:
I noticed that this doesn't do anything if clicked. We should keep that behavior. This part of the header sits directly above the tournament viewer menu. To prevent annoying user error, we don't want them to accidentally click a "go home" button when they just wanted to see the tournament overview.